src/public/main.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let registrationData;
+let occupations;
+let stateOptions;
+let occupationList = document.getElementById('occupationList');
+let statesList = document.getElementById('statesList');
+
```

Comment:
**registrationData** holds all of the data received from the fetch call on page load. This data is then separated into the **occupations** and **stateOptions** variables to be assigned to their respective drop-down menus, which are selected via **occupationList** and **statesList**.